validator/src/main.rs

```diff
@@ -1203,6 +1204,13 @@ pub fn main() {
                 .multiple(true)
                 .help("Path to accounts shrink path which can hold a compacted account set."),
         )
+        .arg(
+            Arg::with_name("snapshot_dir")
+                .long("snapshot-dir")
```

Comment:
> I have added option --snapshots (similar --accounts).

Looking at this now and re-reading what you wrote in the PR description, I completely agree that `--snapshots` is a nicer name for this parameter.   Sorry for the wrong turn
```suggestion
            Arg::with_name("snapshots")
                .long("snapshots")
```